### DISCUSSION TITLE
Meeting JP Hotfix & Improvements 07/06/2023

### DISCUSSION THREAD

**Discussion by mathisfchx** [org-SCAN/website#425] [Meetings]
- permissions des roles : expliciter les catégories et les permissions
- nouveau manuel très simple (pour remplacante JP).
- 1er bug : Force Run ne semble rien faire.
- 2ème bug : Add Event error (je l'ai aussi)
- 3ème : Add List error (je l'ai pas encore)
- Ajouter la suppression ? (soft ou hard ?)  : surtout pour un fields
- Ajouter l'export sur CSV : TODO
- echelle de 1 à 10 au lieu de 1 à 100 dans le weight d'un field

**Comment by mathisfchx** [comment]
Expliquer mieux comment ajouter un event à une personne et comment ajouter des relations multiples

**Comment by johnsilverme** [comment]
Good
